homeassistant/components/dyson/sensor.py

```diff
@@ -24,10 +25,22 @@
     "filter_life": "mdi:filter-outline",
     "carbon_filter_state": "mdi:filter-outline",
     "hepa_filter_state": "mdi:filter-outline",
+    "combi_filter_state": "mdi:filter-outline",
     "humidity": "mdi:water-percent",
     "temperature": "mdi:thermometer",
```

Comment:
Humidity and temperature sensors should use device class and not set custom icons. This can be changed in a future PR.

Reply:
I'll handle that in another PR